### ISSUE TITLE
ARM Stub method signature set_debug_traps mismatch with README

### ISSUE THREAD

**Issue by zachbryant** [real-xinu/xkdb#2]
The readme recommends using `void set_debug_traps(void)` while the ARM stub defines the method as `void set_debug_traps(int n, void *p)`. What are `n` and `p`, and how can we properly initialize Xinu for xkdb?

**Comment by ammaraskar** [comment]
Good catch, unfortunately the ARM stub isn't very well tested. I don't recall if it was ever in a finished state. If you can, try using the galielo/x86/i386 version for now. I'll look into fixing the ARM stub tonight.
